content/cloud/noops-sql-db.md

```diff
@@ -69,34 +67,41 @@ name = "c3"
 sqlite_databases = ["default"]
 ```
 
-The database will be created when the Spin application is deployed. The data persists Spin application invocations and updates. 
+The above configuration will last the lifespan of the application.
```

Comment:
This is not correct - the user could change the configuration and redeploy.

Again, the original sentence aimed to communicate the lifecycle of the _database_ (and the data).

Reply:
Thanks; reverted in next push.